core/sup_tree_core/version_upgrade_task_queue.ex

```diff
@@ -64,10 +78,18 @@ defmodule AntikytheraCore.VersionUpgradeTaskQueue do
   # Public API
   #
   defun gear_updated(gear_name :: v[GearName.t]) :: :ok do
-    :ok = GenServer.cast(__MODULE__, {:gear_updated, gear_name})
+    GenServer.cast(__MODULE__, {:upgrade, {:gear_updated, gear_name}})
```

Comment:
Now that `{:upgrade, _}` is in place, let's remove redundancy in the inner (original) atom:

- `{:upgrade, {:gear, gear_name}}`
- `{:upgrade, :core}`

should suffice.

Reply:
fixed in 9492230